pi_mqtt_gpio/modules/bme280.py

```diff
@@ -0,0 +1,195 @@
+from pi_mqtt_gpio.modules import GenericSensor
+
+#DEVICE = 0x76 # Default device I2C address
+#bus = smbus.SMBus(2)
+
+REQUIREMENTS = ("smbus",)
+
+CONFIG_SCHEMA = {
+    "i2c_bus_num": {"type": "integer", "required": True, "empty": False},
+    "chip_addr": {"type": "integer", "required": True, "empty": False},
+    }
+
+SENSOR_SCHEMA = {
+    "type": dict (
+        type="string",
+        required=False,
+        empty=False,
+        default="temperature",
+        allowed=["temperature", "humidity", "preassure"],
+    ),
+}
+
+class Sensor(GenericSensor):
+    """
+    Implementation of Sensor class for the LM75 temperature sensor.
```

Comment:
Docstring needs updating to be relevant to the 'BME280' module

Reply:
correct with b7e75f044c7eb475d6ea36f648c60d8c37c73bf1